specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-03-01-preview/authorization-PermissionsCalls.json

```diff
@@ -0,0 +1,189 @@
+            "schema": {
+              "$ref": "#/definitions/PermissionGetResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List permissions for resource group": {
+            "$ref": "./examples/Permissions_ListForResourceGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Authorization/permissions": {
+      "get": {
+        "tags": [
+          "Permissions"
+        ],
+        "operationId": "Permissions_ListForResource",
+        "description": "List all permissions the caller has for a resource.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./common-types.json#/parameters/ResourceProviderNamespaceParameter"
+          },
+          {
+            "$ref": "./common-types.json#/parameters/ResourceTypeParameter"
+          },
+          {
+            "$ref": "./common-types.json#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns an array of permissions.",
+            "schema": {
+              "$ref": "#/definitions/PermissionGetResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List permissions for resource": {
+            "$ref": "./examples/Permissions_ListForResource.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PermissionGetResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "./common-types.json#/definitions/Permission"
+          },
+          "description": "An array of permissions."
+        },
+        "nextLink": {
```

Comment:
>"nextLink" [](start = 8, length = 10)

Mark this property as readOnly: true because is your service setting this value, in general, any value set by your service that is not meant to be set by an end-user should be marked as readOnly : true

Reply:
Fixed here and a few other places, thank you